src/common/actions/snaps.js

```diff
@@ -0,0 +1,55 @@
+import 'isomorphic-fetch';
+
+import { checkStatus, getError } from '../helpers/api';
+import { conf } from '../helpers/config';
+
+const BASE_URL = conf.get('BASE_URL');
+
+export const FETCH_SNAPS = 'FETCH_REPOSITORIES';
+export const FETCH_SNAPS_SUCCESS = 'FETCH_SNAPS_SUCCESS';
+export const FETCH_SNAPS_ERROR = 'FETCH_SNAPS_ERROR';
+
+export function fetchUserSnaps(owner) {
+  return (dispatch) => {
+    const url = `${BASE_URL}/api/launchpad/snaps/list?owner=${owner}`;
+
+    dispatch({
+      type: FETCH_SNAPS
+    });
+
+    return fetch(url, {
+      headers: { 'Content-Type': 'application/json' },
+      credentials: 'same-origin'
+    })
+      .then(checkStatus)
+      .then(response => {
+        return response.json().then(result => {
```

Comment:
I think we're standardising on parenthesising the signature part of arrow functions taking just one parameter.